.changeset/afraid-trees-marry.md

```diff
@@ -0,0 +1,10 @@
+---
+'@sl-design-system/shared': patch
```

Comment:
I'm not entirely sure how this applies to the `shared` package? I see you removed a single line there? Can you elaborate?